packages/tasks/src/tasks/image-text-to-text/data.ts

```diff
@@ -58,6 +58,10 @@ const taskData: TaskDataCustom = {
 			description: "Strong image-text-to-text model made to understand documents.",
 			id: "mPLUG/DocOwl1.5",
 		},
+		{
+			description: "Strong image-text-to-text model made for information retrieval from documents.",
+			id: "vidore/colpali",
```

Comment:
This is not really an image-text-to-text model, right? It's used for indexing data in a vector database IIUC

Reply:
Yes, this one seems more of `image-feature-extraction`